CorsixTH/Lua/rooms/operating_theatre.lua

```diff
@@ -104,6 +104,30 @@ function OperatingTheatreRoom:isOperating()
   return false
 end
 
+--! Builds the second operation action (i.e. with the surgeon whose we
+--! see the back). Called either when the operation starts or when the
+--! operation is resumed after interruption caused by the picking up of
+--! the second surgeon.
+--!param multi_use (action): the first operation action (built with via buildTableAction1()).
+--!param operation_table_b (OperatingTable): slave object representing the operation table.
+local function buildTableAction2(multi_use, operation_table_b)
+  local num_loops = math.random(2, 5)
+
+  local loop_callback_use_object = --[[persistable:operatring_theatre_use_callback]] function(action)
```

Comment:
Hmm, not sure if worth pursuing, but I wonder if `operatring` could be changed to `operating` in these callback names without much hassle (unless the spelling was intentional).